File: src/Image4ioSDK/Cls/Image4ioException.vb
Imports System

Public Class Image4ioException
    Inherits Exception

    Public Sub New(ByVal errorMessage As String, ByVal errorCode As String)
        MyBase.New(errorMessage)
    End Sub
End Class


Public Class ExceptionCls
    Public Shared Function CreateException(errorMesage As String, errorCode As String) As Image4ioException
        Return New Image4ioException(errorMesage, errorCode)
    End Function
End Class

File: Image4ioSDK/Image4ioSDK/Cls/Image4ioException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Image4ioSDK
{
    //public class Image4ioException : Exception
    //{
    //    public Image4ioException(string errorMessage, int errorCode) : base(errorMessage) { }
    //}

    //public class ExceptionCls
    //{
    //    public static Image4ioException CreateException(string errorMesage, int errorCode)
    //    {
    //        return new Image4ioException(errorMesage, errorCode);
    //    }
    //}
    public class Image4ioException : Exception
    {
        public Image4ioException(string errorMesage, int errorCode) : base(errorMesage) { }
    }

}
